ministryofjustice/peoplefinder
# CT-3261 Use apikey instead of basic auth

## Description
Upgrade authentication method from Basic to API on peoplefinder / sendgrid integration - to send Emails through Sendgrid after 9th December. Can use same method, just update username/password to API information according to docs ultimately realised.

## Self-review checklist


### Screenshots
Changed username and password based on this:
![image](https://user-images.githubusercontent.com/22935203/99991335-cfde2880-2dac-11eb-85ac-811b9b803f04.png)
Email confirmation still working, so based on that assuming that it's working. tested on dev, testing demo/staging too.

### Related JIRA tickets
https://dsdmoj.atlassian.net/browse/CT-3261

### Deployment
New secrets in deployment

### Manual testing instructions
Try and send any email from Peoplefinder and see if it works, e.g. login confirmation email.
